examples/blinky/CMakeLists.txt

```diff
@@ -4,9 +4,25 @@ set(CMAKE_TOOLCHAIN_FILE ${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/stm32_gcc.cmake
 project(stm32-blinky C ASM)
 set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
 
-find_package(CMSIS COMPONENTS STM32L0 STM32F1 STM32F4 REQUIRED)
-find_package(HAL COMPONENTS STM32L0 STM32F1 STM32F4 RCC GPIO CORTEX REQUIRED)
-# Find all drivers:
+# F4 example enabled by default
+option(F4_EXAMPLE "Compile F4 example" ON)
```

Comment:
It's strange to  have an option with no effect. Add the required `if(F4_EXAMPLE)`

Reply:
I agree, make it ON by default, but allow to turn it OFF using the adequate if(F4_EXAMPLE)

Reply:
Updated